java/code/src/com/redhat/rhn/manager/task/TaskManager.java

```diff
@@ -54,6 +54,23 @@ private TaskManager() {
         return m.execute(params);
     }
 
+    /**
+     *  Get the channel package list for a channel
+     * @param channel channel info
+     * @param start index of first element
+     * @param pageSize how many elements to fetch
+     * @return the iterator
+     */
+    public static DataResult<PackageDto> getChannelPackageIds(Channel channel, long start, int pageSize) {
```

Comment:
Nitpick: I find it a little bit confusing to talk about IDs in the method signature, then return Dtos. I get that the Dto doesn't contain anything but the ID at this point, but I would welcome an extra refactoring (ideally in a separate commit) to return, say, `DataResult<Long>`, so that type better reflects content.

Reply:
Don't we iterate over this result and use all of the values? Not only the id?
In this case I would rename the function.

Seems we call elaborate() on the object to get the rest of the values.

Reply:
I renamed the method to `getChannelPackageDtos`.